package.json

```diff
@@ -47,5 +46,7 @@
     "rimraf": "^3.0.2",
     "serve": "^12.0.0"
   },
-  "dependencies": {}
```

Comment:
There's already a toast library installed and this is the wrong place to install it.